README.md

```diff
@@ -49,8 +48,7 @@ To generate an application, you only need the Raygun gem and network connectivit
 
 To run your new application's specs or fire up its server, you'll need to meet these requirements.
 
-* PostgreSQL 9.x with superuser 'postgres' with no password (`createuser -s postgres`)
-* PhantomJS for JavaScript testing (`brew install phantomjs`)
+* PostgreSQL with superuser 'postgres' with no password (`createuser -s postgres`)
```

Comment:
There are much newer versions of postgres available, and the version number changes fairly often. Rather than try to keep this updated, I just removed the version number.

PhantomJS is no longer required since we now use chromedriver. See: https://github.com/carbonfive/raygun-rails/pull/90